### PR TITLE
Bitloops install

### PR DESCRIPTION
**Pull request by Varagos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,34 +4,36 @@
...
 
-
 ### liveSync
+
 ![liveSync](https://github.com/bitloops/bitloops/blob/f2c77aad338bca10c1338d38e807a0da665fe9f8/docs/assets/Bitloops-helloWorld.gif)
 
-*Create and update strongly-typed HTTP/2 REST APIs in seconds and use them instantly as RPCs on your client code with just a single line of code (only available for TypeScript projects currently)*
+_Create and update strongly-typed HTTP/2 REST APIs in seconds and use them instantly as RPCs on your client code with just a single line of code (only available for TypeScript projects currently)_
 
+_Create and update strongly-typed HTTP/2 REST APIs in seconds and use them instantly as RPCs on your client code with just a single line of code_
```
Comment (sardounis) [inline README.md]: It seems here is an unnecessary duplication.

Comment (sardounis) [inline README.md]: It seems here is an unnecessary duplication.

File: docs/Install.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+```bash
+curl --location --request POST 'http://localhost:3005/bitloops/request' \
+--header 'workflow-id: 7dcd2115-4536-48a4-97c0-26cea7f21768' \
+--header 'node-id: c2115d23-f4da-46bb-aa86-0af8397cc4c7' \
+--header 'environment-id: development' \
+--header 'workflow-version: v1' \
+--header 'authorization: anonymous' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "workspaceId": "db24bb48-d2e3-4433-8fd0-79eef2bf63df"
```
Comment (danias) [inline docs/Install.md]: Remove our workspaceId

File: docs/Install.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+}'
+```
+
+To start an HTTP tunnel forwarding your bitloops-rest
+
+Follow [ngrok setup](https://dashboard.ngrok.com/get-started/setup), run this next
```
Comment (danias) [inline docs/Install.md]: Remove ngrok

File: mongo-seed/environments.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "_id": "0876c3ea-bbff-474c-8cda-7124af0d0b15",
+    "name": "Docker",
+    "description": "Docker Development environment",
+    "workspaceId": "db24bb48-d2e3-4433-8fd0-79eef2bf63df"
```
Comment (danias) [inline mongo-seed/environments.json]: remove

**Review by danias — CHANGES_REQUESTED** [review]

**Review by danias — DISMISSED** [review]

**Review by danias — APPROVED** [review]

**Pull request merged**